giflite/src/components/Gif.vue

```diff
@@ -1,103 +1,103 @@
 <template>
   <div class="gif">
-      <div class="card" @mouseover="mouseOver" @mouseleave="mouseLeave" @click="download">
-        <Slider
-          :class="{ isShowing: downloaded, hidden: !downloaded }"
-          class="slider"
-          v-if="downloaded && showSlider"
-          :sendToChat="sendToChat" 
-          :changeUserLikes="changeUserLikes"
-          :expand="expand"
-          :url="url"
-          :parent="parent"
-        />
-        <span class="download-icon" :class="{ isShowing: !downloaded, hidden: downloaded }">
-          <img src="../assets/download.png">
-        </span>
-        <div class="imgDiv" :style="widthStyle">
-          <img :src="imgTagUrl" alt="image place" :style="gifStyle">
-        </div>
-
-        <Modal
-          v-if="showModal"
-          :url="url"
-          :sendToChat="sendToChat"
-          :changeUserLikes="changeUserLikes"
-          :parent="parent"
-          @close="showModal = false"
-        ></Modal>
+    <div class="card" @mouseover="mouseOver" @mouseleave="mouseLeave" @click="download">
+      <Slider
+        :class="{ isShowing: downloaded, hidden: !downloaded }"
+        class="slider"
+        v-if="downloaded && showSlider"
+        :sendToChat="sendToChat"
+        :changeUserLikes="changeUserLikes"
+        :expand="expand"
+        :url="url"
+        :parent="parent"
+      />
+      <span class="download-icon" :class="{ isShowing: !downloaded, hidden: downloaded }">
+        <img src="../assets/download.png">
+      </span>
+      <div class="imgDiv" :style="widthStyle">
+        <img :src="imgTagUrl" :style="gifStyle">
```

Comment:
where is alt?

Reply:
I think there is no need to alt . If gif has no url so there is no component to be rendered . so it would be so kind if user sees for example one string instead of a not loaded gif .